From f6d304bbda61b43c7ae10f24f0d455515102354e Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 26 Aug 2014 14:12:37 -0400
Subject: [ethernet] igb: remove unnecessary break after return

Message-id: <1409062359-13063-80-git-send-email-sassmann@redhat.com>
Patchwork-id: 89191
O-Subject: [RHEL7.1 PATCH 79/81] igb: remove unnecessary break after return
Bugzilla: 1091121
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Fabian Frederick <fabf@skynet.be>

Signed-off-by: Fabian Frederick <fabf@skynet.be>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 2a79febdc28e93425f30ca2ebc491185b5b333bc)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/igb/e1000_82575.c b/drivers/net/ethernet/intel/igb/e1000_82575.c
index 72b454c..236a618 100644
--- a/drivers/net/ethernet/intel/igb/e1000_82575.c
+++ b/drivers/net/ethernet/intel/igb/e1000_82575.c
@@ -579,7 +579,6 @@ static s32 igb_get_invariants_82575(struct e1000_hw *hw)
   break;
  default:
   return -E1000_ERR_MAC_INIT;
-  break;
  }
 
  /* Set media type */
-- 
1.7.1